source/array-buffer.js

```diff
@@ -46,6 +46,7 @@ const resizeArrayBuffer = (contents, length) => {
 		return contents;
 	}
 
+	// eslint-disable-next-line n/no-unsupported-features/es-syntax
 	const arrayBuffer = new ArrayBuffer(length, {maxByteLength: getNewContentsLength(length)});
```

Comment:
`ArrayBuffer` resizing is not supported in Node 18.
However, we check for support before calling this function.